htdocs/societe/class/societe.class.php

```diff
@@ -478,6 +478,12 @@ class Societe extends CommonObject
 	 */
 	public $tva_intra;
 
+	public $tva_npr;
```

Comment:
There is no reason to have such property on the societe class. There is no such fields in database.